githubapp/errors.go

```diff
@@ -24,6 +26,12 @@ const (
 	MetricsKeyHandlerError = "github.handler.error"
 )
 
+var (
+	// HandlerRecoverStackDepth is the max depth of stack trace to recover on a
+	// handler panic.
+	HandlerRecoverStackDepth = 32
```

Comment:
Is there a reason this is `32`? Or is that just a reasonable best guess?

Reply:
It's a guess. A limit is required because of how the standard library reads stack frames and 32 seems like a good balance between not being too big but having enough context to understand where the panic came from. For this, I copied the value from `hatpear` and when I wrote that, I _probably_ copied what `pkg/errors` used, but I don't remember.